android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -243,13 +242,12 @@ public CustomTwilioVideoView(ThemedReactContext context) {
         // Start the thread where data messages are received
         dataTrackMessageThread.start();
         dataTrackMessageThreadHandler = new Handler(dataTrackMessageThread.getLooper());
-
     }
 
     // ===== SETUP =================================================================================
 
     private VideoFormat buildVideoFormat() {
-        return new VideoFormat(VideoDimensions.CIF_VIDEO_DIMENSIONS, 15);
```

Comment:
this is why images were always being captured as 352 x 288

Reply:
What?? I can't believe this is the default with no option to change it from the RN side. Whoever else is using this library must have a very different use case from ours, if all they need is low-res 15fps video.

Reply:
Couldnt believe it myself and thought the same haha